apps/wallet-dashboard/components/dialogs/staking/views/StakedInfo.tsx

```diff
@@ -38,6 +38,13 @@ export function StakedInfo({ validatorAddress, accountAddress }: StakedInfoProps
                     value={formatPercentageDisplay(apy, '--', isApyApproxZero)}
                     fullwidth
                 />
+                <KeyValueInfo
+                    keyText="Commission"
+                    value={commission}
+                    fullwidth
+                    tooltipText="The charge imposed by the validator for their staking services."
+                    tooltipPosition={TooltipPosition.Top}
```

Comment:
to keep consistence :pray: 
```suggestion
                    tooltipPosition={TooltipPosition.Right}
```

Reply:
@cpl121 fyi
it was changed to Right back, because label was changed from "Validator commission" to "Commission"